src/Behat/Testwork/Tester/Cli/StrictController.php

```diff
@@ -34,6 +36,11 @@ final class StrictController implements Controller
      */
     private $strict;
 
+    /**
+     * @var StrictHandler
+     */
+    private $strictHandler;
+
```

Comment:
As above, I'd suggest using strict types for new code (but leave the existing props with `@var` declarations for now - modernising old code is a separate concern IMO).

```suggestion
    private StrictHandler $strictHandler;
```